internal/service/cloudprovideraccess/resource_cloud_provider_access_authorization.go

```diff
@@ -299,10 +301,6 @@ func authorizeRole(ctx context.Context, client *admin.APIClient, d *schema.Resou
 	if err != nil {
 		return diag.FromErr(fmt.Errorf("error cloud provider access authorization %s", err))
 	}
-	role, _, err := client.CloudProviderAccessApi.GetCloudProviderAccessRole(ctx, projectID, roleID).Execute()
-	if err != nil {
-		return diag.FromErr(fmt.Errorf("error cloud provider access authorization read after authorization %s", err))
-	}
```

Comment:
reverts temporary changes that were made in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2536